### DISCUSSION TITLE
Build TinyUSB for RP2040 using FreeRTOS without "CFG_TUSB_OS" redefined warning

### DISCUSSION THREAD

**Discussion by huyle028** [hathach/tinyusb#1834] [Q&A]
Hi all,

How to build TinyUSB for RP2040 using FreeRTOS without "CFG_TUSB_OS" redefined warning when the CFG_TUSB_OS macro is defined as OPT_OS_PICO in hw\bsp\rp2040\family.cmake ?

Many thanks in advance,
Huy LE

**Comment by hathach** [comment]
should be fixed now
